index.js

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs';
+
+const data = fs.readFileSync('words_alpha.txt', 'utf8');
+const words = data.split('\r\n');
+const ALPHABET = ['a', 'b', 'c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'];
+
+function areAlphabetical(words){
```

Comment:
if i want to know what a function does quickly i should be able to tell from the name. this function is called areAlphabetical but it's not telling me if words is alphabetical, it's returning the longest alphabetical word.
So I would call this "getLongestAlphabeticalWord".